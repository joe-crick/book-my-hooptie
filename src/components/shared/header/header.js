import React from "react";
import styles from "./header.less";
import { translate } from "react-i18next";

export const Header = ({ t: translate }) => (
  <header className={styles.header}>
    <h1 className={styles.title}>{translate("logo.title")}</h1>
  </header>
);

export default translate()(Header);
