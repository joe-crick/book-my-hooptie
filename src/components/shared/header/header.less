@import "../../../style-resources/mixins.less";

.header {
    background-color: @slick-purple;
    margin: @flush;
    padding: @surround-pad;
}

.title {
    color: @almost-white;
    font-size: @large-font;
}