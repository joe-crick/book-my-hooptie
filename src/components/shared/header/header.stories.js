import React from "react";
import { storiesOf } from "@storybook/react";
import { i18nMock } from "test/test-utils";
import { Header } from "./header";

storiesOf("Header", module).add("default", () => <Header t={i18nMock} />);
