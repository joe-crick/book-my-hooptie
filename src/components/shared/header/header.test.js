import React from "react";
import { Header } from "./header";
import renderer from "react-test-renderer";
import { i18nMock } from "test/test-utils";

describe("Header", () => {
  it("should render", () => {
    const component = renderer.create(<Header t={i18nMock} />);
    let header = component.toJSON();
    expect(header).toMatchSnapshot();
  });
});
