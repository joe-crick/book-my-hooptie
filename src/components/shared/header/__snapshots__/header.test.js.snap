// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<header
  className={undefined}
>
  <h1
    className={undefined}
  >
    logo.title
  </h1>
</header>
`;
